lib/cpp/src/thrift/server/TSimpleServer.cpp

```diff
@@ -88,7 +88,7 @@ void TSimpleServer::serve() {
       string errStr = string("Some kind of accept exception: ") + tx.what();
       GlobalOutput(errStr.c_str());
       continue;
-    } catch (string s) {
+    } catch (const string& s) {
```

Comment:
I don't know of any code in thrift that intentionally throws a string.  Throwing a string (or anything without a vtable really) is certainly a poor practice.